README.md

```diff
@@ -38,13 +38,7 @@ rake db:seed OR rake db:setup (which also sets up the entire db)
 # edit database.yml to point to your postgresql database
 cp config/database.yml.example config/database.yml
 
-cp config/app_config.yml.example config/app_config.yml
-```
-
-## Generate Secret Token
-
-```
-bundle rake secret > secret_token
+cp .env.example .env
 ```
```

Comment:
Does it have to be a dot-file? I feel like new collaborators may not see the file for a while (or, maybe have a line in README saying that all the setup stuff is in .env)